rules/eslint/style/walmart.js

```diff
@@ -78,6 +80,8 @@ module.exports = {
     "quote-props": 0,
     // specify whether double or single quotes should be used
     "quotes": [2, "double"],
+    // require identifiers to match the provided regular expression
+    "id-match": 0,
```

Comment:
Do those not get caught by camelCase? I thought this was for people who want to enforce really specific things like `_`
prefixing and the like.

https://github.com/walmartlabs/eslint-config-defaults/blob/feature/bring-up-to-1.0/rules/eslint/style/walmart.js#L10